okta/resource_okta_app_oauth.go

```diff
@@ -734,7 +734,7 @@ func buildAppOAuth(d *schema.ResourceData) *okta.OpenIdConnectApplication {
 			InitiateLoginUri:       d.Get("login_uri").(string),
 			LogoUri:                d.Get("logo_uri").(string),
 			PolicyUri:              d.Get("policy_uri").(string),
-			RedirectUris:           convertInterfaceToStringSetNullable(d.Get("redirect_uris")),
```

Comment:
Shouldn't we have some check for uniqueness of redirect_uris? Thoughts?

Reply:
@garmishr I feel like now that the ordering is not modified (is maintained correctly) that any further inspection isn't necessary. If the Okta API doesn't like the input parameters then it will error out and the operator can then go and make corrections to their config.